### PR TITLE
Improve schedule YAML file to fix sle-15 install for ppc64le

### PR DESCRIPTION
**Pull request by Dawei-Pang**
Add fix_sle_15gm_multi_disks for ppc64le sle-15 installation.

- Verification run: http://openqa.qa2.suse.asia/tests/overview?distri=sle&version=15&build=GM&groupid=30

### CODE REVIEW

**Comment by Dawei-Pang** [comment]
@simonlm @badboywj @alice-suse Would you please help to review? Thanks!

**Comment by simonlm** [comment]
LGTM

**Comment by badboywj** [comment]
looks good

File: schedule/sle-perf/hana_install.yaml [hunk 1]
```diff
@@ -75,6 +75,7 @@ conditional_schedule:
         - installation/partitioning_firstdisk
       ppc64le:
         - installation/partitioning_smalldisk_storageng
+        - '{{fix_sle_15gm_multi_disks}}'
```
Comment (b10n1k) [inline schedule/sle-perf/hana_install.yaml:78]: This schedules `installation/partitioning_smalldisk_storageng` twice. is that right?

Comment (Dawei-Pang) [reply schedule/sle-perf/hana_install.yaml:78]: Thanks for your querying! yes, calling the function twice to workaround different between SLE15 and other product.

Comment (Dawei-Pang) [reply schedule/sle-perf/hana_install.yaml:78]: > This schedules `installation/partitioning_smalldisk_storageng` twice. is that right?

Thanks for your querying! yes, calling the function twice to workaround different between SLE15 and other product.

**Comment by alice-suse** [comment]
LGTM. @Julie-CAO @nanzhg FYI about yaml scheduled applied by hana perf.

**Pull request merged**